_docs/getting_started/install.md

```diff
@@ -34,6 +34,25 @@ Install The Lounge:
 $ sudo npm install -g thelounge
 ```
 
+## CentOS / Fedora / RHEL
+
+### Step 1:
+
+Install the requirements using yum(If you are using a newer Fedora version, please use dnf instead):
```

Comment:
Maybe add a Soave between yum and (? Also, “newer version of fedora, please use dnf”) maybe?